plugins/home/src/homePageComponents/Toolkit/Content.tsx

```diff
@@ -35,8 +35,12 @@ const useStyles = makeStyles(theme => ({
   },
   label: {
     marginTop: theme.spacing(1),
+    minWidth: '64px',
+    maxWidth: '72px',
+    width: 'min-content',
     fontSize: '0.9em',
     lineHeight: '1.25',
+    overflowrap: 'break-word',
```

Comment:
Shouldn't this be `overflowWrap`?

Reply:
1+

Reply:
Thank you for the review,  adjusted and pushed the changes!